index.html

```diff
@@ -64,6 +64,22 @@ <h1>Patient History</h1>
 
     <div id="patient-info"></div>
     <div id="immunization-history"></div>
+    <div id="AllergyIntoleranceContent"></div>
+    <div id="MedicationStatementContent"></div>
```

Comment:
Not available in Epic R4 UCSDI. Only Medication and MedicationRequest (which you have) are.

https://fhir.epic.com/Documentation?docId=patientfacingfhirapps&section=Appendix